### PR TITLE
[Hotfix] Fix perf. drop for SWDEV-256648

### PR DESCRIPTION
**Pull request by solaslin**
resolves #SWDEV-256648

Summary of proposed changes:
- update the tensile-tag to the hotfix, Tensile [PR#1120](https://github.com/ROCmSoftwarePlatform/Tensile/pull/1200)

### CODE REVIEW

**Comment by solaslin** [comment]
Please wait for CQE to test and give us OK before we merge into rocm-3.10.x

**Review by zaliu — APPROVED** [review]

**Review by bragadeesh — APPROVED** [review]

**Pull request merged**
